docs/src/examples/components/Tooltip/Types/TooltipControlledExample.shorthand.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react'
+import { Button, Tooltip } from '@stardust-ui/react'
+
+class TooltipControlledExample extends React.Component {
+  state = { open: false }
+
+  handleOpenChange = (e, { open }) => {
+    console.log(`Tooltip requested to change its open state to "${open}".`)
+    this.setState({ open })
+  }
+
+  render() {
+    return (
+      <Tooltip
+        open={this.state.open}
+        onOpenChange={this.handleOpenChange}
+        trigger={<Button icon="expand" />}
+        content={{ content: 'This is a controlled Tooltip' }}
+      />
+    )
+  }
+}
```

Comment:
Can be reworked with knobs...

Reply:
https://github.com/stardust-ui/react/blob/727e9c2915ca8450e1e030ab37d1b7bc12e65615/docs/src/examples/components/Popup/Variations/PopupExamplePosition.shorthand.tsx#L7

Reply:
I'll rework the example, but I am a bit worried, that we are not showing how the onOpenChange handler should be used in this case.. :\

Reply:
Knobs expose `set` callback as second argument:

```
const [open, setOpen] = useBoolean()
//
onOpenChange={(e, data) => setOpen(data.open)}
```

Reply:
Ok, I'll reworked the example like this, as I see this as a rather important thing for the clients to be aware of.